src/commands/protocols/common/handle-protocol-message-command.js

```diff
@@ -127,6 +127,21 @@ class HandleProtocolMessageCommand extends Command {
                 this.blockchainModuleManager.getR0(blockchain),
                 getAsk(),
             ]);
+        const blockchainAssertionSizeInKb = blockchainAssertionSize / BYTES_IN_KILOBYTE;
+        if (
+            this.config.maximumAssertionSizeInKb > 0 &&
```

Comment:
Should this check be updated ? what happens if user sets maximumAssertionSizeInKb to 0 ?

Reply:
This check won't be performed if 0 is set in configuration. It's part of the spec in the document